src/Data/Fin/Properties.agda

```diff
@@ -365,6 +390,31 @@ toℕ-inject₁ (suc i) = cong suc (toℕ-inject₁ i)
 toℕ-inject₁-≢ : ∀ {n}(i : Fin n) → n ≢ toℕ (inject₁ i)
 toℕ-inject₁-≢ (suc i) = toℕ-inject₁-≢ i ∘ ℕₚ.suc-injective
 
+inject₁≢suc : ∀ {n} → (i : Fin n) → inject₁ i ≢ suc i
```

Comment:
Hmm I'm not sure this one is that generally applicable? Do you have a use case for it?

Reply:
I have in old commits but having refactored my code and cleaned up unusued imports, I realise I'm now not using it at all - so probably not worth having.